src/Libraries/RevitNodes/Application/Document.cs

```diff
@@ -62,6 +66,71 @@ public static Document Current
             get { return new Document(DocumentManager.Instance.CurrentDBDocument); }
         }
 
+        /// <summary>
+        /// Purge unused Elements from the model. This node is not able to purge materials and material assets
+        /// </summary>
+        /// <param name="deepPurge">Similar to clicking the purge button multiple times in Revit.</param>
+        /// <returns></returns>
+        public string PurgeUnused(bool deepPurge)
+        {
+            //The internal GUID of the Performance Adviser Rule 
+            const string purgeGuid = "e8c63650-70b7-435a-9010-ec97660c1bda";
```

Comment:
rename var to something a bit more descriptive, maybe check what the name of this rule is ?